.travis.yml

```diff
@@ -8,8 +8,8 @@ env:
   - TRAVIS_NODE_VERSION="6"
   - TRAVIS_NODE_VERSION="8.10"
   - TRAVIS_NODE_VERSION="10"
-  - TRAVIS_NODE_VERSION="11"
```

Comment:
Could you undo these removals? Node 6 is also EOL, but I consider removing a Node.js version from CI a breaking change

Reply:
Sure, done! Thanks for the fast review.